### ISSUE TITLE
Append to an existing contextMenu?

### ISSUE THREAD

**Issue by smelcher** [swisnl/jQuery-contextMenu#770]
Is it possible to grab an existing context menu and append an additional menu to it?

**Comment by bbrala** [comment]
Yes, you don't need to grab or mutate an existing menu for this. Use `build` instead of a static `items` object: it runs fresh every time the menu is about to open, so you can merge in extra items at that point from wherever you like:

```js
$.contextMenu({
    selector: '.my-trigger',
    build: function ($trigger, e) {
        return { items: $.extend({}, baseItems, extraItems) };
    }
});
```

See the `build` docs and the dynamic-create demo. There's no dedicated "append items to a live menu" API today, but `build` gets you the same result.
